guides/policy/azure-authenticator/parseAzureAuthenticator.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+    $(document).ready(function () {
+
+        $('#encodedToken').on('input', tokenInputChanged);
+
+        $('#tblInput input').on('input', hostIdInputChanged);
+    });
+
+    let hostId;
+    let encodedToken;
+
+    function hostIdInputChanged() {
+        hostId = $("#txtHostId").val();
+        updatePolicy();
+    }
+
+    function tokenInputChanged() {
+        encodedToken = $("#encodedToken").val();
+        updatePolicy();
+    }
+
+    function updatePolicy() {
+        let output = "";
+        if (!hostId) {
+            output = "host ID is missing";
+        } else if (!encodedToken) {
+            output = "Azure AD token is missing";
+        } else {
+            output = getPolicyFromToken(encodedToken, hostId);
+        }
+        $("#output").html(output);
```

Comment:
do you plan to show here other things besides authn policy? in not then I suggest the id will reflect this